docs/source/virtual_rainforest/main_simulation.md

```diff
@@ -0,0 +1,65 @@
+# Virtual Rainforest simulation flow
+
+This document describes the main simulation flow of the Virtual Rainforest model. This
+flow will now be set out step by step.
+
+## Loading configuration files
+
+As a first step, configuration files are loaded in and validated. These files are of
+`toml` format and are found based on paths provided by the user. When these files are
+loaded in they are validated to ensure that they are valid `toml`, and that no tags are
+repeated. In cases where a required tag is not present in any file, the validation will
+fail unless a default value for it has been set. Further details can be found in the
+[configuration documentation](./core/config.md).
+
+## Grid creation
+
+The next step is creating a grid, which defines the area and resolution of the
+simulation. The [grid documentation](./core/grid.md) describes this object further. This
+grid is then used to create an empty `Data` object of the correct size to describe the
+simulation.
+
+## Loading and validation of input data
+
+This `Data` object now needs to be populated with data. The data used in the simulation
+is stored in a set of data files which are specified in the configuration. This data
```

Comment:
This will change, I think. I think `Grid` should be part of a `Config` object, and `Data` should just be data. But that's for a future PR.